### PR TITLE
Made blastoff.c and blastoff.go, requesting review

### PR DESCRIPTION
**Pull request by GeneralNotSteve**

### CODE REVIEW

**Review by JordanRose-Ari — APPROVED** [review]
Like the comments for yourself, looks good.

**Review by jacobGolden — COMMENTED** [review]
LGTM

**Review by jacobGolden — APPROVED** [review]

File: blastoff.c [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+	const int START = 10;
+
+	for (int i = START; i >=0; i--)
+		printf("%d\n", i);
+	
+
+	printf("BLAST OFF");
+
+	return 0;
+	//YOU NEED A RETURN
```
Comment (defreez) [inline blastoff.c:14]: Using pseudocode comments like this is a great way to develop! I do the same thing, but they should be removed from the final product.

**Review by defreez — COMMENTED** [review]
LGTM

**Pull request merged**